openlibrary/templates/account/import.html

```diff
@@ -31,7 +31,7 @@ <h3>$_("Import from Goodreads")</h3>
           </div>
         </form>
       </div>  
-      <div>
+      <div class="import-export-readinglog">
```

Comment:
We should use BEM notation here. In BEM notation, this would be import-export__readinglog 
e.g.  import-export is the block, and readinglog is an element of block...

https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide